lib/search_config.rb

```diff
@@ -22,63 +35,16 @@ def index_names
     content_index_names + auxiliary_index_names
   end
 
-  def content_index_names
-    elasticsearch["content_index_names"] || []
-  end
-
-  def auxiliary_index_names
-    elasticsearch["auxiliary_index_names"] || []
```

Comment:
By dinamically creating these methods, you lost the `|| []` would this affect something?